tests/integration/targets/mso_schema_site_anp/tasks/main.yml

```diff
@@ -444,24 +295,6 @@
     - nm_non_existing_site is not changed
     - nm_non_existing_site.msg == "Site 'non-existing-site' is not a valid site name."
 
-# USE A TEMPLATE WITHOUT ANY SITE
-- name: Add ANP to Template without any site associated (normal mode)
-  cisco.mso.mso_schema_site_anp:
-    <<: *mso_info
-    schema: '{{ mso_schema | default("ansible_test") }}_2'
-    site: 'azure_{{ mso_site | default("ansible_test") }}'
-    template: Template 4
-    anp: ANP_4
-    state: present
-  ignore_errors: true
-  register: nm_no_site_associated
-
-- name: Verify cm_no_site_associated and nm_no_site_associated
-  ansible.builtin.assert:
-    that:
-    - nm_no_site_associated is not changed
-    - nm_no_site_associated.msg == "No site associated with template 'Template4'. Associate the site with the template using mso_schema_site."
-
```

Comment:
Should we move those tests to on-prem site vs removing them?

Reply:
It is part of the `on-prem` tasks.